### DISCUSSION TITLE
OSPP Mentorship 2023 Challenge - for #2451

### DISCUSSION THREAD

**Discussion by juntao** [WasmEdge/WasmEdge#2526] [Show and tell]
# Introduction

For those who want to apply for OSPP mentorship for https://github.com/WasmEdge/WasmEdge/issues/2451, this is a pretest for the application.

# Requirements

1 [Install WasmEdge with the Rust toolchain](https://wasmedge.org/docs/develop/rust/setup)

2 Please sign up for a free developer account at OpenAI and deploy the PR code review bot to one of your own GitHub repos. 

https://github.com/flows-network/github-pr-review

3 Then, make changes to your own bot’s source code so that it only reviews a specific type of source code files that are used in your repo (eg Java or Python source code). You can also modify the prompt to create better reviews for that file type. 

# Deadline

Please refer to [the OSPP 2023 timeline](https://summer-ospp.ac.cn/?lang=en) for the final deadline for valid submissions, which is June 4th, UTC+8.

We look forward to reviewing your submissions and thank you for your interest.

**Comment by redoC-A2k** [comment]
I have a doubt for the llm pretest , for the bot to review only some specific programming language files,   can I identify the programming language by their file extension ? (For example .py - means python programming language or .java for java programming language )
Or some other way has to be used ?

**Reply by juntao** [reply]
Of course you can. You can add any logic to the bot's source code as you see fit.

**Comment by k7Xm9pQr** [comment]
Hi! Here is my showcase:
https://github.com/iGxnon/Code-Review-Pull-Request/pull/3
